src/main/java/com/google/neighborgood/servlets/EditTaskServlet.java

```diff
@@ -124,6 +124,22 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
       return;
     }
 
+    // Get the task overview from the form input
```

Comment:
Personally task overview sounds too much like task detail, would task title reflect better what we're trying to achieve with this field?

Reply:
I think overview is totally different from detail. Overview is just a general review or summary of a certain task, such as "Help me buy some food from a nearby supermarket", but task detail will include the shopping list and requirements. So I think overview is a good name choice here.

Reply:
Sounds good then! Would it be possible to make this distinction more explicit in the add task form then? Perhaps adds something that suggests the overview is meant to be one sentence only. having the input fields differ in size (or have a max set of characters allowed for the overview) might also help emphasize this distinction.

Reply:
Ah I would try to add this feature in a separate PR. It's a little bit hard to make such kind of distinction obvious and I will try to figure out how to fix this

Reply:
I have created an issue #87 for this problem